packages/react/src/components/input/input.stories.tsx

```diff
@@ -37,3 +42,29 @@ export const Disabled = storyOf(Input, 'disabled', [true, false], {
   labelProp: 'defaultValue',
 });
 Disabled.argTypes = omit<InputProps>('disabled');
+
+interface InputWithFieldLabelProps extends InputProps {
+  id: string;
+  label: string;
+  helperText: string;
+}
+
+export const WithFieldLabel = ({
+  id,
+  label,
+  helperText,
+  ...restInputProps
+}: InputWithFieldLabelProps) => (
+  <>
+    <FieldLabel htmlFor={id}>
+      {label}
+      {helperText && <HelperText>{helperText}</HelperText>}
+    </FieldLabel>
+    <Input {...restInputProps} id={id} />
```

Comment:
My ideal API would be

```tsx
<Field>
  My label
  <Input />
</Field>
```

We could then either use the provided `id` and `htmlFor` or auto generate one by default.

That complicates the styling though, we could use `gap` if we require the `<Input>` to be inside the `<Field>` which honestly makes more sense in my opinion.